### PR TITLE
feat(invoices/conversions): moved the conversions from detail page to…

### PR DESCRIPTION
**Pull request by LindaAchilles**
… edit/create page and changed the position of the fields

#57

### CODE REVIEW

File: src/coda/apps/invoices/views/update.py [hunk 1]
```diff
@@ -13,14 +13,41 @@
...
+            exchange_rate = Decimal(request.POST["exchange_rate"])
+            invoice.add_conversion(exchange_rate, conversion_currency)
+            repository.update(invoice)
+
+        if selected_currency == home_currency.code or not exchange_rate_str:
+            for currency in list(invoice.conversions().keys()):
+                invoice.remove_conversion(currency)
+            repository.update(invoice)
+
+        invoice_id, errors = save_invoice(request, invoice_id=invoice.id, existing_invoice=invoice)
```
Comment (SvenMarcus) [inline src/coda/apps/invoices/views/update.py]: You are using repository.update() and save_invoice() here, so two ways of saving invoices to the database. Please stick to the save_invoice function

File: tests/invoices/test_add_conversion_view.py [hunk 2]
```diff
@@ -5,59 +5,112 @@
...
+    client: Client,
+) -> None:
     creditor = CreditorId(modelfactory.creditor().id)
-    expected = domainfactory.invoice(creditor=creditor, positions=())
-    expected.add_conversion(Decimal("2.0"), Currency.JPY)
-    expected.id = repository.create(expected)
+    invoice = domainfactory.invoice(creditor=creditor, positions=())
+    invoice.id = repository.create(invoice)
+    invoice.add_conversion(Decimal("2.0"), Currency.JPY)
+    repository.update(invoice)
```
Comment (SvenMarcus) [inline tests/invoices/test_add_conversion_view.py]: Why do you create (=save) the invoice first, then make some changes and then update it? Why not create a fully configured from the start?

File: tests/invoices/test_add_conversion_view.py [hunk 3]
```diff
@@ -5,59 +5,112 @@
...
+@pytest.mark.django_db
+@pytest.mark.usefixtures("logged_in")
+def test__saved_invoice_with_conversion__invoice_currency_is_changed_to_home_currency__conversion_is_deleted(
+    client: Client,
+) -> None:
+    GlobalPreferences.set_home_currency(Currency.EUR)
+    fr = funding_request()
+    publication_position = domainfactory.publication_position(
+        fr.publication.id, currency=Currency.JPY
+    )
```
Comment (SvenMarcus) [inline tests/invoices/test_add_conversion_view.py]: I believe we only need this position in the test because invoices currently determine their currency by the position currencies, right? 
We should consider storing the currency in the invoice once instead of having a full money object in every position.

**Review by SvenMarcus — CHANGES_REQUESTED** [review]
Please address the SonarQube issues and also tackle the edge case where an institution may change its home currency

File: tests/invoices/test_add_conversion_view.py (hunk 3 above)
Comment (LindaAchilles) [reply tests/invoices/test_add_conversion_view.py]: I created a new issue for this: #71
